### PR TITLE
fix(vscode): dart frog version parsing when update available

### PR DESCRIPTION
**Pull request by jsgalarraga**
## Status

**READY**

## Description

Fix dart frog version parsing in vscode extensión when an update is available

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by erickzanardo — APPROVED** [review]

File: extensions/vscode/src/test/suite/utils/cli-version.test.ts [hunk 1]
```diff
@@ -23,15 +23,37 @@ suite("readDartFrogCLIVersion", () => {
...
+      cpStub.execSync.returns(encodedDartFrogVersionCommandResult);
 
-    assert.strictEqual(cliVersion.readDartFrogCLIVersion(), "0.3.7");
-  });
+      assert.strictEqual(cliVersion.readDartFrogCLIVersion(), "0.3.7");
+    }
+  );
+
+  test(
+    "returns the version of Dart Frog CLI installed in the user's system " +
```
Comment (alestiago) [inline extensions/vscode/src/test/suite/utils/cli-version.test.ts:41]: I think we can group these tests in: "returns the version of Dart Frog CLI installed in the user's system".

File: extensions/vscode/src/test/suite/utils/cli-version.test.ts [hunk 2]
```diff
@@ -23,15 +23,37 @@ suite("readDartFrogCLIVersion", () => {
...
+      "when new version is available",
+    () => {
+      const dartFrogVersionCommandResult =
+        "0.3.7\n\n" +
```
Comment (alestiago) [inline extensions/vscode/src/test/suite/utils/cli-version.test.ts:45]: **Nit:**

Use backticks (`) for multiline string instead.

**Review by alestiago — COMMENTED** [review]
Thanks for contributing! 💙

File: extensions/vscode/src/test/suite/utils/cli-version.test.ts (hunk 2 above)
Comment (jsgalarraga) [reply extensions/vscode/src/test/suite/utils/cli-version.test.ts:45]: Didn't find this online! 😓 
Thanks!

File: extensions/vscode/src/test/suite/utils/cli-version.test.ts [hunk 3]
```diff
@@ -23,15 +23,35 @@ suite("readDartFrogCLIVersion", () => {
...
-  test("returns the version of Dart Frog CLI installed in the user's system", () => {
-    const dartFrogVersionCommandResult = "0.3.7\n";
-    const encoededDartFrogVersionCommandResult = new TextEncoder().encode(
-      dartFrogVersionCommandResult
-    );
-    cpStub.execSync.returns(encoededDartFrogVersionCommandResult);
+  suite(
+    "returns the version of Dart Frog CLI installed in the user's system",
+    () => {
+      test("when is last version", () => {
```
Comment (alestiago) [inline extensions/vscode/src/test/suite/utils/cli-version.test.ts:29]:
```suggestion
      test("when on latest version", () => {
```

**Review by alestiago — APPROVED** [review]

**Review by alestiago — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]

**Pull request merged**